test/OpenTelemetry.Instrumentation.Process.Tests/ProcessMetricsTests.cs

```diff
@@ -158,7 +158,7 @@ private static double GetValue(Metric metric)
         {
             if (metric.MetricType.IsGauge())
             {
-                sum += metricPoint.GetGaugeLastValueDouble();
+                sum += metricPoint.GetGaugeLastValueLong();
```

Comment:
Why this change? Also, the if-else condition seems incorrect. Were you trying to do something similar to this: 

https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/test/OpenTelemetry.Instrumentation.Runtime.Tests/RuntimeMetricsTests.cs#L187-L198

Reply:
process.memory.usage is using .NET API WorkingSet64 which is `long`.
Previously, we used `double?` to cache the value. Thus, the change.

I've removed the unnecessary branching.